src/ORM/Behavior/Translate/EavStrategy.php

```diff
@@ -492,6 +496,7 @@ protected function bundleTranslatedFields(EntityInterface $entity): void
                 $contents[$i]->setNew(false);
             } else {
                 $translation['model'] = $this->_config['referenceName'];
+                unset($translation['foreign_key IS']);
```

Comment:
Some strange code let to this

Reply:
In the past when using `'foreign_key' => null` the ORM auto converted to `foo IS null` in the query, but later we started requiring to explicitly use the `IS` in the key.

At that point `'foreign_key' => $key` was changed to `'foreign_key IS' => $key` in the `$field` array few lines above leading to `'foreign_key IS'` to be used as an entity field.